cli/flox-rust-sdk/src/flox.rs

```diff
@@ -63,11 +64,25 @@ impl Flox {}
 
 #[derive(Debug, Clone, Deserialize, Serialize, Default)]
 pub struct Features {
+    /// Which matching logic to use when searching for packages
+    #[serde(default)]
+    pub search_strategy: SearchStrategy,
+    #[serde(default)]
+    pub use_catalog: UseCatalog,
     /// Whether to enable services
     #[serde(default)]
     pub services: bool,
 }
 
+#[derive(Copy, Clone, Debug, Serialize, Deserialize, derive_more::Deref)]
+pub struct UseCatalog(bool);
+
+impl Default for UseCatalog {
+    fn default() -> Self {
+        UseCatalog(true)
+    }
+}
+
```

Comment:
nonblocking: refering back to my comment in the other pr i think we should do the same for services

Reply:
I left a note about it in the commit:

> I looked at pulling `Features.services` out to it's own type at the same time, to make it easier to change the default later, but whilst the default is `false` it violates a clippy rule, so it doesn't seem worth doing at the moment:
> 
> - https://github.com/flox/flox/pull/1712#discussion_r1666985785
> - https://rust-lang.github.io/rust-clippy/master/index.html#/derivable_impls